plugin-server/src/cdp/consumers/cdp-cyclotron-worker.consumer.ts

```diff
@@ -24,13 +24,10 @@ export class CdpCyclotronWorker extends CdpConsumerBase {
         const invocationResults = await runInstrumentedFunction({
             statsKey: `cdpConsumer.handleEachBatch.executeInvocations`,
             func: async () => {
-                // NOTE: In the future this service will never do fetching (unless we decide we want to do it in node at some point)
```

Comment:
I'd still keep the note to say "this is primarily used for e2e testing and isn't actually used in production"